package.json

```
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "3.7.0",
+    "@docusaurus/plugin-vercel-analytics": "^3.7.0",
```

```suggestion
    "@docusaurus/plugin-vercel-analytics": "3.7.0",
```

Should probably pin this to match the other docusaurus packages.